core/blockchain.go

```diff
@@ -1148,6 +1165,7 @@ func (bc *BlockChain) SnapSyncHvm(btcTipHeader *chainhash.Hash, hvmTipHeader *ty
 		Headers: headersToAdd,
 	}
 
+	log.Info(fmt.Sprintf("hVM snap sync adding %d headers", len(msgHeaders.Headers)))
```

Comment:
Not blocking, but how noisy is this one?

Wonder if maybe it should be something other than Info (unless it really isn't too noisy).

Reply:
This one only prints once ever so going to leave in since it could be helpful debugging if anyone has an issue with hVM header-only mode state post-snap sync